.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ env:
   RSYNC_HOST: ijzer
```

Comment:
in this build script, can you rename references of CN-WS to WaTEM/SEDEM (CN is an extension now)